packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -257,14 +266,55 @@ const TransactionHeader = memo(
           width={20}
           onSelect={e => dispatchSelected({ type: 'select-all', event: e })}
         />
-        <Cell value="Date" width={110} />
-        {showAccount && <Cell value="Account" width="flex" />}
-        <Cell value="Payee" width="flex" />
+        <Cell
+          value="Date"
+          id="date"
+          width={110}
+          icon={field === 'date' && ascDesc}
+          onClick={e => onSortTable(e)}
+        />
+        {showAccount && (
+          <Cell
+            value="Account"
+            id="account"
+            width="flex"
+            icon={field === 'account' && ascDesc}
+            onClick={e => onSortTable(e)}
```

Comment:
This would let you completely avoid the DOM access in the event handler

```suggestion
            value="Account"
            width="flex"
            icon={field === 'account' && ascDesc}
            onClick={() => onSortTable("account")}
```